src/index.js

```diff
@@ -196,9 +196,11 @@ export default class SauceLabs {
                     ),
                     responseType: 'json'
                 })
-
                 return response.body
             } catch (err) {
+                if (propName === 'createJob') {
```

Comment:
connecting this behavior to a special endpoint feels odd to me. I am sure we would be interested in the error response for other endpoints to, what do you think?

Reply:
I think it would be better to show err response for all endpoints but I found some duplicate messages when testing around other API, just like
```
./bin/sl updateJob cb-onboarding xxxxx "{}"
Error: Failed calling updateJob: Response code 404 (Not Found), Not Found
```
Is it fine to return such err like this?